src/components/homePage/Testimonials.vue

```diff
@@ -1,13 +1,32 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <h2>Testimonials</h2>
+    <!-- SIDE SCROLLER -->
+    <vueper-slides>
+      <vueper-slide 
```

Comment:
Would it be possible to control the layout further with this library? Like, the contents of each slide?

Reply:
Agreed, it would be nice to control the inner connect with a child or something